Juliusz Podraza - lab01

vatros67 opened the pull request:
Zadania na 4.5

vatros67 committed:
Zadania na 4.5

DocentSzachista commented on projekt_do_pracy_na_labach/fiszki_backend/Dockerfile:
```diff
@@ -1,2 +1,38 @@
+#docker run backend
+
+#2
+#docker run backend -p 8000:8000
```
Tutaj jesteś niekonsekwetny, stworzyłeś obraz `docbuild` a nie `backend`. Prosiłbym o poprawienie :P

DocentSzachista commented on projekt_do_pracy_na_labach/fiszki_backend/Dockerfile:
```diff
@@ -1,2 +1,38 @@
+CMD ["python", "main.py"] 
+#1
+#docker build --tag docbuild
+#docker run backend
```
To samo co komenttarz poniżej / powyżej

DocentSzachista reviewed (commented):
Jak na razie 4.5 (O ile mi poprawisz to co opisałem w komentarzu :P)

vatros67 committed:
Zadanie na 4.5 poprawione

DocentSzachista committed:
Update lab-01.md

DocentSzachista committed:
Wykomentowanie zmiennej BACKEND_URL, dodanie portu który jest udostępniany

Teraz docker-compose będzie działał. 

TLDR:
- docker compose tworzy defaultową siec, po której jeżeli udostepnicie jakikolwiek port, będzie on dostępny pod adresem localhost
- Aby sposób ze zmienną zadziałał, wystarczy podmienić link na http://localhost:8000 
- W domyśle zadania było aby użyć nazwy serwisu i żeby aplikacja frontendowa nawiązała połączenie po DNS'ie. (Ale coś nie pykło z apką flutterową).

DocentSzachista committed:
Dodanie instrukcji numer dwa do ocenienia

DocentSzachista committed:
Poprawki

DocentSzachista committed:
Merge pull request #30 from DocentSzachista/boss-man-consulting

Dodanie labów 2

DocentSzachista committed:
Drobne poprawki w słowie pisanym i obrazek

DocentSzachista committed:
Dodanie zadanek do lab 3, niedokończona wersja

DocentSzachista committed:
Dodanie opisów

DocentSzachista committed:
Aktualizacja rozpiski

DocentSzachista commented:
Chcesz próbować na 5.0 czy zamykać?

DocentSzachista committed:
Merge pull request #31 from DocentSzachista/boss-man-consulting

Drobne poprawki w słowie pisanym i lab 3 (lab3 niedokończone jeszcze)

vatros67 committed:
Merge pull request #1 from DocentSzachista/master

Lab2

DocentSzachista commented:
Zamykam z 4.5
